.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: required
 
+before_install:
+  - sudo apt-get update
```

Comment:
@ananyo2012 My tests were failing on Travis CI, I googled the error message and that was the solution. Maybe I shouldn't have made the change then. I didn't run `bundle update`